Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -151,9 +177,13 @@ public void TestNoDependencyCollectionDiagnosticSourceNoResponseClose()
                 module.ProfileQueryEndpoint = FakeProfileApiEndpoint;
                 module.Initialize(this.config);
 
-                var url = new Uri("https://www.bing.com/");
+                var url = new Uri(LocalhostUrl);
                 HttpWebRequest request = WebRequest.CreateHttp(url);
-                HttpWebResponse response = (HttpWebResponse)request.GetResponse();
+
+                using (new LocalServer(LocalhostUrl))
+                {
+                    request.GetResponse();
+                }
```

Comment:
Please add a comment indicating DiagnosticSource won't collect dependency if response is not disposed.